### ISSUE TITLE
Rdio OAuth2 endpoint down?

### ISSUE THREAD

**Issue by sastraxi** [rdio/api#162]
Starting about 1.5h ago, we are no longer able to access the Rdio OAuth2 API. As a result our app is not working (java.net.SocketTimeoutException: Read timed out).

Attaching traceroute to services.rdio.com, I believe the problem is that the service is inaccessible. Can you confirm?

```
traceroute to services.rdio.com (8.18.203.22), 30 hops max, 60 byte packets
 1  75.127.69.129 (75.127.69.129)  0.636 ms  0.833 ms  1.065 ms
 2  72.9.236.113 (72.9.236.113)  0.576 ms  0.578 ms  0.569 ms
 3  xe-7-2-1.mpr3.atl6.us.zip.zayo.com (128.177.104.133)  0.523 ms  0.536 ms  0.
517 ms
 4  ae8.cr2.iah1.us.zip.zayo.com (64.125.20.173)  13.221 ms  13.217 ms  13.208 m
s
 5  v42.ae29.cr2.sjc2.us.zip.zayo.com (64.125.31.160)  60.185 ms  60.152 ms  60.
387 ms
 6  xe-0-2-0.mpr1.sjc37.us.zip.zayo.com (64.125.25.125)  60.625 ms  60.661 ms  6
0.656 ms
 7  xe-1-0-0.mpr1.sfo2.us.zip.zayo.com (64.125.25.130)  60.898 ms  60.881 ms  60
.884 ms
 8  * * *
 9  * * *
10  * * *
11  * * *
12  * * *
13  * * *
14  * * *
15  * * *
16  * * *
17  * * *
18  * * *
19  * * *
20  * * *
21  * * *
22  * * *
23  * * *
24  * * *
25  * * *
26  * * *
27  * * *
28  * * *
29  * 208.184.34.66.IPYX-068466-ZYO.above.net (208.184.34.66)  62.049 ms !X *
```

**Comment by sastraxi** [comment]
``` java

    private static final String TOKEN_URL = "https://services.rdio.com/oauth2/token";
    private static final String AUTH_URL = "https://www.rdio.com/oauth2/authorize";
    private static final String API_URL = "https://services.rdio.com/api/1/";
```

**Comment by mlrsmith** [comment]
We believe this was fixed around 9.15am PST (outage we think began at around 7.50am).

We're investigating root cause and trying to avoid this happening again, but for now the system should be back to normal.

**Comment by sastraxi** [comment]
Thanks for the quick reply Michael!
